apps/src/aiEvaluation/aiFeedbackApi.ts

```diff
@@ -0,0 +1,39 @@
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
```

Comment:
Hooray for making this more general and giving it its own place to live! 

nit: can we name this file `aiInteractionFeedbackApi` just because it matches the model more closely and gives a little clue that this isn't feedback given _by_ AI (well, it does in my brain anyway).